### DISCUSSION TITLE
[0.7] LocalResource update signals

### DISCUSSION THREAD

**Discussion by soundprojects** [leptos-rs/leptos#3313] [Q&A]
Hi

Is it possible in 0.7+ to capture outer signals within a LocalResource and update them? I have an asynchronous closure and I want the result of the operation to set some signals in the outer scope.

**Comment by gbj** [comment]
It sounds like a bad idea to me, as it runs some arbitrary side effects during what would otherwise be a derived computation.

Why not `spawn_local`?

(But also, have you tried? It probably works. And if you've tried and had problems, it would be worth sharing some code.)
